web-components/src/components/block-content/BlockContent.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import BlockContent from '@sanity/block-content-to-react';
+
+import { UnderlineTooltip } from '../tooltip/UnderlineTooltip';
+import { Box, SxProps, Theme } from '@mui/material';
+
+export type SanityBlockContent = any[] | any; // copied from Sanity's types
+export type SanityBlockOr<T> = T | SanityBlockContent;
+
+const CustomBlockContent: React.FC<{
+  className?: string;
+  content: SanityBlockContent;
+  tooltipText?: string;
+  noYMargin?: boolean;
+  /** we can't pass `sx` directly, so use wrapper to target elements */
+  sxWrap?: SxProps<Theme>;
```

Comment:
I was hoping Git handled this rename better... I moved the logic out of the main `index` file just because I had several `index` files open in my editor and prefer named components. Can easily revert if needed. 

The only new part here is `sxWrap` - used to target the rendered content via `sx`

Reply:
yes, it is also my preference to avoid index files so that they contain code... at most to export